fabfile/fabric_settings.py.default.py

```diff
@@ -1,39 +1,51 @@
+"""
+Welcome to the `fabric_settings.py` file!
+
+Below is a field that will fill in the Ansible configurations with your desired
+values. Once you're finished, save and then do a fab command. To see the list of
+fab commands, enter in `fab -l`.
+
+If you're filling this out for an existing Yurt project, the only necessary fields
+to fill in are `git_repo` and `project_name`. The other options are only necessary
```

Comment:
Actually, do we even need project name? We might just need the github repo and that's it.